### PR TITLE
fix struct tags and test in ldaputil

### PR DESCRIPTION
**Pull request by fairclothjm**
In sdk/helper/ldaputil/config.go, two of the struct tags are incomplete and now fail `vet`.

Because this is persisted data, the practical best we can do is fix them to be CamelCase, lest we break installations. We should do this though to avoid someone quickly “fixing” them by just adding the missing ".

### CODE REVIEW

**Review by vinay-gopalan — APPROVED** [review]

File: sdk/helper/ldaputil/config.go [hunk 1]
```diff
@@ -385,11 +383,13 @@ type ConfigEntry struct {
...
 	RequestTimeout           int    `json:"request_timeout"`
 
-	// This json tag deviates from snake case because there was a past issue
-	// where the tag was being ignored, causing it to be jsonified as "CaseSensitiveNames".
+	// These json tags deviate from snake case because there was a past issue
+	// where the tag was being ignored, causing it to be jsonified as "CaseSensitiveNames", etc.
 	// To continue reading in users' previously stored values,
 	// we chose to carry that forward.
-	CaseSensitiveNames *bool `json:"CaseSensitiveNames,omitempty"`
+	CaseSensitiveNames *bool  `json:"CaseSensitiveNames,omitempty"`
```
Comment (jasonodonnell) [inline sdk/helper/ldaputil/config.go:390]: Seems weird that this is the only config that has `omitempty`, any idea why?

**Review by jasonodonnell — APPROVED** [review]
LGTM!

**Review by pcman312 — APPROVED** [review]

File: sdk/helper/ldaputil/config.go [hunk 2]
```diff
@@ -385,11 +383,13 @@ type ConfigEntry struct {
...
+	ClientTLSCert      string `json:"ClientTLSCert"`
+	ClientTLSKey       string `json:"ClientTLSKey"`
```
Comment (calvn) [inline sdk/helper/ldaputil/config.go:392]: Should we also add a test to cover this? Maybe adding these values to `jsonConfig` which then gets exercised by `TestConfig`.

**Review by calvn — APPROVED** [review]
One suggestion on test coverage, but otherwise looks good.

**Pull request merged**